src/main/java/com/iridium/iridiumskyblock/managers/SchematicManager.java

```diff
@@ -100,17 +101,17 @@ private void pasteSchematic(final Island island, final World world, final Schema
         // Loop all blocks in the schematics layer at the current y level
         for (int x = 0; x < schematic.getLength(); x++) {
             for (int z = 0; z < schematic.getWidth(); z++) {
-                Block block = island.getCenter(world).subtract(schematic.getLength()/2.00, -90, schematic.getWidth()/2.00).add(x, y, z).getBlock();
+                Block block = island.getCenter(world).subtract(schematic.getLength()/2.00, -90, schematic.getWidth()/2.00).add(x, (-90 + yHeightIsland) + y, z).getBlock();
```

Comment:
why doing here `(-90 + yHeightIsland)` im didn't understand could you explain it, why not directly yHeightIsland ?

Reply:
Basically it is created at the height of 90 blocks, we subtract the 90 to return to 0 base and we let the server choose

Reply:
Okay so just do it as `yHeightIsland - 90` :panda:

Reply:
Or directly add it to subracting part will be more better

Reply:
```suggestion
                Block block = island.getCenter(world).subtract(schematic.getLength()/2.00, 0, schematic.getWidth()/2.00).add(x,  yHeightIsland + y, z).getBlock();
```